DNN Platform/Connectors/GoogleAnalytics/Scripts/connector.js

```diff
@@ -105,7 +106,7 @@ define(["jquery", "knockout", "templatePath/scripts/config", "templatePath/scrip
 
                         // Set the isDeactivating flag to true to override the default save behaviour
                         // Temporary workaround until delete functionality on connectors is improved
-                        conn.configurations[3].value("true");
+                        conn.configurations[5].value("true");
```

Comment:
Is there any way to get away from mysterious numbers defining these values?

Reply:
Agreed, It's something I'd also like to fix, but I didn't have time this morning to look into the configuration property in the connector engine. I'm looking at it now to see if we can remove those and replace them with the name of the setting.